SDL/Quintuplet.cu

```diff
@@ -1799,11 +1799,11 @@ __device__ bool SDL::runQuintupletDefaultAlgoBBBB(struct SDL::modules& modulesIn
     float tl_axis_lowEdge_x = tl_axis_x;
     float tl_axis_lowEdge_y = tl_axis_y;
 
-    betaIn = alpha_InLo - SDL::deltaPhi(mdsInGPU.anchorX[firstMDIndex], mdsInGPU.anchorY[firstMDIndex], tl_axis_x, tl_axis_y);
+    betaIn = alpha_InLo - SDL::phi_mpi_pi(atan2f(tl_axis_y,tl_axis_x)-mdsInGPU.anchorPhi[firstMDIndex]);
```

Comment:
If you still have the phi function in Hit.cuh, maybe you should use that here instead of just applying atan2f directly. Just so that the code is more readable. Same comment for a few other places in the code.

Reply:
Thank you for the comments Gavin! I agree that since I still have the phis, it makes no difference whether I use the phi function or the atan directly. I'm now using the atan expression because I feel like there is no clear meaning for this variable atan(tl_axis_y, tl_axis_x). It's not a "phi" for something but more of a temporary calculation. Do you think it make sense to leave it here?